src/components/Q&A/AnswerComp.jsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import AnswerReport from './AnswerReport.jsx';
+import {API_KEY} from '../../config/config.js';
+const axios = require('axios');
+
+
+class AnswerComp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answerList: this.props.answerList,
```

Comment:
I think this does not have to be a part of state, unless you need to dynamically change it inside this component. It seems from where it is used on lines 42-43, you're just referencing answerLists, not modifying it so instead of making it part of state, I would just reference it. Example:
(line 42) -> var aList = this.props.asnwerList

Reply:
Good catch, went a head and changed that too since I don't modify it past that.